alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/src/test/java/org/alfresco/transformer/LibreOfficeTransformationIT.java

```diff
@@ -163,6 +177,15 @@ public LibreOfficeTransformationIT(final Pair<TestFileInfo, TestFileInfo> entry)
                 allTargets("quick.ods", spreadsheetTargets),
                 allTargets("quick.xls", spreadsheetTargets),
                 allTargets("quick.xlsx", spreadsheetTargets),
+
```

Comment:
What about document targets & presentation targets (in addition to spreadsheet targets) ?

Reply:
this should be resolved now, however, I am not sure which section dita goes into. i put it in spreadhseetTargets for now. see
https://github.com/Alfresco/alfresco-transform-core/pull/252/commits/8f1d240cc70c602b0f099d1c3eff384bf744f8b7

somewhere along the way my added code broke the previously working local tests so I  looking into that now

Reply:
Fixed ? 

Do we have tests for x39 (or is it now x37 ?) new supported transforms ?

Reply:
it is 37 now :(

Reply:
It's a good start - let's get this merged. I'm sure part 2 will add more :-)